### PR TITLE
Check tutorials

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: tutorials/settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+rootProject.name = "tutorials"
```
Comment (igordmn) [inline tutorials/settings.gradle.kts:1]: 1. Maybe we put script not under 'tutorials' folder, but under scripts/check-tutorials? So `tutorials` remains clean.
2. I think it is better to add gradlew or gradlew.bat in the folder.
3. Also we need to add `build` and `.gradle` folder into .gitignore

File: tutorials/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    val proc = ProcessBuilder("bash", "./gradlew", "build")
+      .directory(snippet.tempDir)
+      .redirectOutput(ProcessBuilder.Redirect.PIPE)
+      .redirectError(ProcessBuilder.Redirect.PIPE)
+      .start()
+    proc.waitFor(5, TimeUnit.MINUTES)
+    if (proc.exitValue() != 0) {
+      System.err.println("Error in snippet at ${snippet.file}:${snippet.lineNumber}")
+      println(proc.inputStream.bufferedReader().readText())
+      println(proc.errorStream.bufferedReader().readText())
```
Comment (igordmn) [inline tutorials/build.gradle.kts:66]: We need to fail gradle task if some tutorial contains error. So CI can tell us if something wrong.

**Comment by olonho** [comment]
PTAL

File: tutorials/settings.gradle.kts (hunk 1 above)
Comment (igordmn) [reply tutorials/settings.gradle.kts]: You forgot to move `gradle/wrapper` folder :)

**Review by Estabrook1988 — APPROVED** [review]

**Review by igordmn — APPROVED** [review]

**Pull request merged**